chapters/git-cmdline.Rmd

```diff
@@ -788,6 +808,14 @@ $ git commit -a -m "Update dracula plot"
  rewrite results/dracula.png (99%)
 ```
 
+The Git commands we've covered so far (`git add`, `git commit`, `git diff`)
+represent the tasks you perform in a basic Git workflow in a local repository 
+(Figure \@ref(fig:git-cmdline-remote)a).
+
+```{r git-cmdline-remote, echo=FALSE, fig.cap="FIXME: Do we need a figure similar to this (removing some of the commands that aren't relevant to this chapter)?"}
```

Comment:
Did you want to update the figure caption?

Reply:
Yikes, thanks for catching that.